conformance/runner/run_cv_bundle.py

```
@@ -2236,6 +2236,326 @@ def run_chainstate(gate: str, fixture: dict[str, Any], rust: ClientCmd, go: Clie
+        after = line[colon + 1 :]
+        first = after.find("`")
+        if first < 0:
+            continue
+        after_first = after[first + 1 :]
+        second = after_first.find("`")
+        if second < 0:
+            continue
+        value = after_first[:second].strip()
+        if value:
+            return value
+    raise ValueError(f"missing key: {key}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)